README.md

```diff
@@ -8,6 +8,12 @@ Use SDK in Lens extension:
 import { LensPlatformClient } from "lens-platform-sdk";
 import { Component } from "@k8slens/extensions";
 
+// axios is used by lens-platform-sdk
+import axios from "axios";
+
+// Use NodeJS HTTP adapter to get around CORS issues
+axios.defaults.adapter = require("axios/lib/adapters/http");
```

Comment:
Can we do this is in sdk? Like detecting if sdk is used in Electron, and use the adapter.

Reply:
Maybe.. but I feel like it's the responsibility of the library user